app/views/home/_navbar.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id="vertical-nav" class="nav-pf-vertical
+     nav-pf-vertical-with-sub-menus
+     nav-pf-persistent-secondary">
+  <!-- menu -->
```

Comment:
nitpick: not sure if you wanted it in the resulting html, but if its just a template comment, erb comments should be 
```erb
<%# comment goes here %>
```